.vscode/settings.json

```diff
@@ -119,7 +119,6 @@
 		"postcss": "css"
 	},
 	"eslint.format.enable": true,
-	"eslint.packageManager": "yarn",
```

Comment:
Why remove this?

Reply:
We can keep it — but VS Code indicated that it's now handled by default and the setting has been deprecated.

Reply:
https://github.com/microsoft/vscode-eslint/blob/fa396e45eb25087950c81c9a9f9eb6f7d1f2818f/README.md?plain=1#L218